osis-app/src/main/java/com/scality/osis/healthcheck/UtapiHealthIndicator.java

```diff
@@ -26,27 +26,30 @@ public class UtapiHealthIndicator implements HealthIndicator {
 
     @Override
     public Health health() {
-        HttpURLConnection connection = null;
-        try {
-            // check if Utapi Endpoint is reachable
-            connection = (HttpURLConnection) new java.net.URL(appEnv.getUtapiEndpoint()).openConnection();
-            connection.setConnectTimeout(appEnv.getUtapiHealthCheckTimeout());
-            connection.connect();
-        } catch (SocketTimeoutException e) {
-            logger.warn("Failed to connect to Utapi endpoint {} in timeout {}",appEnv.getUtapiEndpoint(), appEnv.getUtapiHealthCheckTimeout());
-            return Health.down()
-                    .withDetail("error", e.getMessage())
-                    .build();
-        } catch (IOException e) {
-            logger.warn("an I/O error occurs while trying to connect {}.",appEnv.getUtapiEndpoint());
-            return Health.down()
-                    .withDetail("error", e.getMessage())
-                    .build();
-        } finally {
-            if (connection != null) {
-                connection.disconnect();
-            }
-        }
-        return Health.up().build();
+        // We want to make sure healthcheck is false even if it is set in application.properties
+        // This will be enabled and potentially refactored with S3C-8266
+
+        // HttpURLConnection connection = null;
+        // try {
+        //     // check if Utapi Endpoint is reachable
+        //     connection = (HttpURLConnection) new java.net.URL(appEnv.getUtapiEndpoint()).openConnection();
+        //     connection.setConnectTimeout(appEnv.getUtapiHealthCheckTimeout());
+        //     connection.connect();
+        // } catch (SocketTimeoutException e) {
+        //     logger.warn("Failed to connect to Utapi endpoint {} in timeout {}",appEnv.getUtapiEndpoint(), appEnv.getUtapiHealthCheckTimeout());
+        //     return Health.down()
+        //             .withDetail("error", e.getMessage())
+        //             .build();
+        // } catch (IOException e) {
+        //     logger.warn("an I/O error occurs while trying to connect {}.",appEnv.getUtapiEndpoint());
+        //     return Health.down()
+        //             .withDetail("error", e.getMessage())
+        //             .build();
+        // } finally {
+        //     if (connection != null) {
+        //         connection.disconnect();
+        //     }
+        // }
+        return Health.down().build();
```

Comment:
This will be refactored in the future as there are some edge cases where not setting the right configuration causes unexpected issues. Hence even if this is not callable as we commented code below, we have commented this code.

Reply:
Can we just delete it, as it stays in the git history forever?